pm.md

```diff
@@ -0,0 +1,173 @@
+`mptcp-flags` in the [settings](https://networkmanager.dev/docs/api/latest/nm-settings-nmcli.html) --
+and [mptcpd](https://mptcpd.mptcp.dev). Here, the focus is on the manual
+configuration, using the [`ip mptcp`](https://man7.org/linux/man-pages/man8/ip-mptcp.8.html)
+command.
+
+#### Endpoints
+
+MPTCP endpoints can be configured with this command:
+
+```sh
+ip mptcp endpoint add <IP address> dev <interface> [ signal | subflow ] [ backup ] [ fullmesh ]
+```
+
+{: .warning}
+It is important to specify the network interface linked to the address by adding
+`dev <interface>`. If not, the routing will probably not be done properly, and
+will require manual configuration, see below:
+[Manual Routing Configuration](#manual-routing-configuration).
+
+One of the following flags needs to be set:
+- `signal`: The endpoint will be announced to each peer via an MPTCP `ADD_ADDR`
+  sub-option. Typically, a server would be responsible for this.
+- `subflow`: The endpoint will be used to create an additional subflow using
+  the given source IP address. A client would typically do this.
+
+Optionally, the following flags can be added next to one of the previous ones:
+- `backup`: Subflows created from this endpoint instruct the peers to only send
+  data on it when all non-backup subflows are unavailable.
+- `fullmesh`: The MPTCP path manager will try to create an additional subflow
+  for each known peer address, using this endpoint as the source IP address. It
+  requires the `subflow` flag, and it is not compatible with the `signal` one.
+
+The IP address is an IPv4 or IPv6 address. The endpoints are netns-aware.
+
+#### Example
+
+- Servers can announce extra IP addresses:
+```sh
+ip mptcp endpoint add 10.2.2.2 dev eth0 signal
+```
+
+- Clients can create additional subflows from a cellular interface, and flag
+  this subflow as "backup", to be used to carry data only if the main path is
+  unavailable:
+```sh
+ip mptcp endpoint add 100.64.1.134 dev usb0 subflow backup
+```
+
+#### Limits
+
+It is also important to make sure the limits are high enough:
+
+```sh
+ip mptcp limits set [ subflows NR ] [ add_addr_accepted NR ]
+```
+
+- `subflows` is the limit of additional created and accepted subflows (paths),
+  for both the client and server sides (default is 2).
+- `add_addr_accepted` is the limit of accepted `ADD_ADDR` -- IP address
+  notification from the other peer -- that will result in the creation of
+  subflows, typically only for the client side (default is 0).
+
+The limits are per MPTCP connection, and netns-aware.
+
+{: .note}
+It is possible to reach the limits with fewer established subflows than
+expected, e.g. when new subflow requests cannot reach the other peer. In case of
+problem, please increase the limits, use `ss -Mai` to check the counters, and
+set the adapted routing or firewall rules to avoid using certain paths between
+specific IP addresses, e.g. in a lab setup with dedicated links, don't use
+default but specific routes.
```

Comment:
Sync with #11 suggestion:

```suggestion
modify the adapted routing or firewall rules to avoid using certain paths between
specific IP addresses. For example, in a lab setup with dedicated links use specific routes 
rather than letting the kernel select the default route.
```